nx.json

```diff
@@ -137,18 +137,23 @@
   "defaultBase": "master",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33074&commitId=3790d75ed13bbb771a92e71fbb4476e66512b2b0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|1| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3290| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>